### ISSUE TITLE
Is there a reason why BrowserHistory was used in the production

### ISSUE THREAD

**Issue by cr05s19xx** [chentsulin/electron-react-boilerplate#1458]
I ran across a few issues when using `createBrowserHistory()` as my production version. The problem was that when I used `history.goBack()`, it routed me to my root drive (I use Windows so that is my C: drive). As my solution, I switched to using `createHashHistory()`.

My question is, is there an advantage of using `createBrowserHistory()` over `createHashHistory()` in the production version? If so, can I get a suggestion on how to fix my problem while still using `createBrowserHistory()`

**Comment by uppe-r** [comment]
Hey @cr05s19xx you can see a comparison between the two (and memory as well) [here](https://github.com/ReactTraining/history#usage). Apart from that, there is also a great [Medium article](https://medium.com/@pshrmn/a-little-bit-of-history-f245306f48dd) that goes deeper into the history module, in which it is referred as well that they create a location from the URL differently:

```js
// Given the following URL
url = 'http://www.example.com/this/is/the/path?key=value#hash'
// a browser history creates the location object:
{
  pathname: '/this/is/the/path',
  search: '?key=value',
  hash: '#hash'
}
// a hash history creates the location object:
{
  pathname: 'hash',
  search: '',
  hash: ''
}
```

As for your issue, maybe try using `history.go(-1)` instead of `history.goBack()` ?

**Comment by amilajack** [comment]
A fix for this was added in `v0.13.3`
